src/components/ImageSlider/index.js

```diff
@@ -50,13 +93,27 @@ class Images extends Component {
     this.setState({ activeIndex: index })
   }
 
-  handlePress = i => () => {
+  handleRightArrow = () => {
+    const { activeIndex } = this.state
+    const { images } = this.props
+    if (activeIndex > images.length - 2) return
+    this.handleChange(activeIndex + 1)
+  }
+
+  handleLeftArrow = () => {
+    console.log('handleLeftArrow')
+    const { activeIndex } = this.state
+    if (activeIndex === 0) return
+    this.handleChange(activeIndex - 1)
+  }
+
+  handlePress = (i) => () => {
```

Comment:
This doesn't match our normal prettier config. Could you just copy the prettier config from another project?